### PR TITLE
Fix scrolling through equations crashing page

### PR DESCRIPTION
**Pull request by DZGoldman**
Fixes https://github.com/natejenkins/socialApp/issues/3740

 The browser's range tracking gets confused inside of katex- rendered  equations. Normally, a cursor inside a non-editable would just be moved out, making it a non-issue, but for this particular case, it crashes the page right away. So for this case, we prevent the default behavior up front and handle it explicitly.

### CODE REVIEW

File: build/squire-raw.js [hunk 1]
```diff
@@ -1698,6 +1698,18 @@ var onKey = function ( event ) {
         return;
     }
 
+    if(event.altKey && [37,39].includes(code)){
```
Comment (dginev) [inline build/squire-raw.js]: image

(adding a comment can't hurt here, right?)

Comment (DZGoldman) [reply build/squire-raw.js]: lmao, touche

**Comment by DZGoldman** [comment]
The came from an intercom report; would be nice to be able to tell the user it's fixed. Given that the scope is very narrow and specific, I'm tempted to merge...

**Comment by apepe** [comment]
go for it and I can tell Kevin is fixed when fix is live

**Pull request merged**